Modules/Nonunit/Review/include/itkScalarChanAndVeseSparseLevelSetImageFilter.hxx

```diff
@@ -38,8 +38,7 @@ ScalarChanAndVeseSparseLevelSetImageFilter<TInputImage, TFeatureImage, TOutputIm
     InputPointType    origin = input->GetOrigin();
 
     // In the context of the global coordinates
-    FeatureIndexType start;
-    this->GetInput()->TransformPhysicalPointToIndex(origin, start);
+    FeatureIndexType start = this->GetInput()->TransformPhysicalPointToIndex(origin);
```

Comment:
It seems many of these could be marked `const` now?

Reply:
Indeed, thanks Matt! Maybe for another PR? (I would like to have this one merged as soon as it's "good enough", because it fixes a few warnings at https://open.cdash.org/index.php?project=Insight)